src/main/resources/assets/some_assembly_required/lang/fr_fr.json

```diff
@@ -0,0 +1,74 @@
+{
+  "itemGroup.some_assembly_required": "Some Assembly Required",
+
+  "some_assembly_required.advancement.obtain_bread_slice.title": "Une question de miche (et) de pain",
+  "some_assembly_required.advancement.obtain_bread_slice.description": "Coupez une miche de pain en tranches",
+  "some_assembly_required.advancement.obtain_toasted_bread_slice.title": "Brave Petit Grille-Pain",
+  "some_assembly_required.advancement.obtain_toasted_bread_slice.description": "Faites griller une tranche de pain",
+  "some_assembly_required.advancement.obtain_sandwich.title": "Débutant Sandwichmancien",
+  "some_assembly_required.advancement.obtain_sandwich.description": "Créez votre premier sandwich",
+  "some_assembly_required.advancement.add_potion_to_sandwich.title": "Le Snack du Sorcier",
+  "some_assembly_required.advancement.add_potion_to_sandwich.description": "Mangez un sandwich avec une potion dedans",
+  "some_assembly_required.advancement.consume_double_decker_sandwich.title": "La Double Dalle",
+  "some_assembly_required.advancement.consume_double_decker_sandwich.description": "Mangez un sandwich double-étage",
+
+  "some_assembly_required.message.bottom_bread": "Il manque la tranche en-dessous !",
+  "some_assembly_required.message.full_sandwich": "Mollo sur les ingrédients !",
+
+  "block.some_assembly_required.sandwiching_station": "Station à Sandwichs",
+  "block.some_assembly_required.sandwich": "Sandwich",
+
+  "item.some_assembly_required.spread": "Pâte à Tartiner",
+  "item.some_assembly_required.bread_slice": "Tranche de Pain",
+  "item.some_assembly_required.toasted_bread_slice": "Tranche de Pain Grillée",
+  "item.some_assembly_required.apple_slices": "Tranches de Pomme",
+  "item.some_assembly_required.golden_apple_slices": "Tranches de Pomme Dorée",
+  "item.some_assembly_required.enchanted_golden_apple_slices": "Tranches de Pomme Dorée Enchantée",
+  "item.some_assembly_required.chopped_carrot": "Haché de Carotte",
+  "item.some_assembly_required.chopped_golden_carrot": "Haché de Carotte Dorée",
+  "item.some_assembly_required.chopped_beetroot": "Haché de Betterave",
+  "item.some_assembly_required.tomato_slices": "Tranches de Tomate",
```

Comment:
I only have small knowledge of french. But is the usage of `de` really appropriate here?
As I understand it it marks possesion, which isn't truely the case here and googling "Haché de Carotte" yields "Carottes Hachées", which sounds more true to me.

Reply:
You're right, it does sound truer language-wise, but while it's phonetically better, my approach allows quantification from MC's object stacking: "un Haché de Carottes", "deux HachéS de Carottes", etc... Literal translation would give "one Chop of Carrots", "two ChopS of Carrots", and so on.
Of course I see no objection on any changes, if others find your suggestion better